client/shipping-label/views/dialog.js

```diff
@@ -2,90 +2,116 @@ import React from 'react';
 import Dialog from 'components/dialog';
 import ActionButtons from 'components/action-buttons';
 import { translate as __ } from 'lib/mixins/i18n';
-import sum from 'lodash/sum';
 import AddressStep from './steps/address';
 import PackagesStep from './steps/packages';
 import RatesStep from './steps/rates';
 import PreviewStep from './steps/preview';
 import { hasNonEmptyLeaves } from 'lib/utils/tree';
 import { sprintf } from 'sprintf-js';
+import isEqual from 'lodash/isEqual';
+import debounce from 'lodash/debounce';
 
-const PrintLabelDialog = ( props ) => {
-	const currencySymbol = props.storeOptions.currency_symbol;
+class PrintLabelDialog extends React.Component {
+	maybeGetNewRates( nextProps ) {
+		const useNormalizedOrigin = nextProps.form.origin.selectNormalized;
+		const useNormalizedDestination = nextProps.form.destination.selectNormalized;
 
-	const getTotalCost = () => {
-		const ratesInfo = props.form.rates;
-		const ratesCost = ratesInfo.values.map( ( rateId, index ) => ratesInfo.available[ index ][ rateId ].rate );
-		return sum( ratesCost ).toFixed( 2 );
-	};
+		// Determine if we need to compare user-entered values, or normalized ones
```

Comment:
Agreed - fixed in https://github.com/Automattic/woocommerce-connect-client/pull/533/commits/c902be65c71b39bc216404ca6604707eca80fec1.